app/src/androidTest/java/com/sample/neuroid/us/NIDSchema.kt

```diff
@@ -57,22 +59,23 @@ class NIDSchema {
             )
         validateSchema(json)
 
-        val application = ApplicationProvider.getApplicationContext<Application>()
-        NIDServiceTracker.sendEventToServer(
-           NIDJobServiceManager.getServiceAPI(),
-            "key_live_suj4CX90v0un2k1ufGrbItT5",
-            application,
-            eventList,
-            object: NIDResponseCallBack {
-                override fun onSuccess(code: Int) {
-                    assertEquals(json, 200, code)
-                }
-
-                override fun onFailure(code: Int, message: String, isRetry: Boolean) {
-                    assert(false)
-                }
-            }
-        )
+        // Commenting out because tests shouldn't send traffic to prod. Will put back in once dev url is available
+//        val application = ApplicationProvider.getApplicationContext<Application>()
```

Comment:
For iOS I was exposing isDeveopment to tests, and changing to dev url based on that

` public static var isDevelopment: Bool = false
`

     static func getCollectionEndpointURL() -> String {
        if (NeuroID.isDevelopment) {
            return "https://receiver.neuro-dev.com/c"
        }
        return "https://receiver.neuroid.cloud/c"
    }

`

Reply:
I think we should discuss how we want to expose that to customers because it was causing confusion for people the difference between our version of development (actual development) and customer development (using their test key)